scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -175,10 +171,9 @@
                   }
 
                   markup += "<tr><td class=\"igv-popover-td\">"
-                          + "<div class=\"igv-popover-name-value\">"
```

Comment:
Ill leave that to you if we need it or not; since we anyway use these classes I don't mind either way!

Reply:
I tested and it looks the same without